libs/metadata.liq

```diff
@@ -61,34 +61,30 @@ def metadata.comment(m)
   m["comment"]
 end
 
-# Extract cover from metadata.
+# Extract cover from metadata. This function implements cover extraction
+# for the following formats: coverart (ogg) and apic (flac, mp3). 
 # @category Source / Track Processing
 # @param m Metadata from which the cover should be extracted.
+# @param ~coverart_mime Default mime type for `"coverart"` data.
 # @method mime MIME type for the cover.
-def metadata.cover(m)
+def metadata.cover(~coverart_mime="image/jpeg", m)
```

Comment:
```suggestion
def metadata.cover(~mime="image/jpeg", m)
```

Reply:
I don't thik that there is any ambiguity here...

Reply:
Hmm okay but let's reword the description then.